build/gen_stub.php

```diff
@@ -1060,7 +1060,7 @@ public function getArgInfoName(): string {
     }
 
     public function getMethodSynopsisFilename(): string {
```

Comment:
Well, not exactly as it doesn't do the whole extension to ``reference/ext/`` mapping which would be great to do and also ignore zend_test so that it can be ran "globally"